tests/modules/vg/deconstruct/main.nf

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { VG_DECONSTRUCT } from '../../../../modules/vg/deconstruct/main.nf'
+
+workflow test_vg_deconstruct {
+    input = [ [ id:'test' ], // meta map
+              [ file(params.test_data['homo_sapiens']['pangenome']['pangenome_fa'], checkIfExists: true)],
```

Comment:
I think you misunderstood `vg deconstruct`. `--path` takes one or several path names in the graph as a string, not a FASTA input.